workspace/resource_notebook.go

```diff
@@ -101,8 +101,10 @@ var mtx = &sync.Mutex{}
 
 // Create creates a notebook given the content and path
 func (a NotebooksAPI) Create(r ImportPath) error {
-	mtx.Lock()
-	defer mtx.Unlock()
+	if r.Format == "DBC" {
```

Comment:
@alexott do we need to check if Format is `AUTO`? This could also be a DBC notebook and run into this race condition.

Reply:
We don't allow `AUTO` as format: https://github.com/databricks/terraform-provider-databricks/blob/master/workspace/resource_notebook.go#L329